test_freqswitch.py

```diff
@@ -25,16 +26,17 @@
 import pylab
 
 sp = pyspeckit.Spectrum('test_freqswitch.fits')
-sp.plotter()
+# sp.plotter()
 pylab.plot(x,y,'k--',drawstyle='steps-mid')
 
 print ("First test - do the FITS version and the input match up?")
-import pdb; pdb.set_trace()
+# import pdb; pdb.set_trace()
```

Comment:
Why *wasn't* this commented out before?  This is a question directed at me - this is weird!